### PR TITLE
chronyc, ntpq: add page

### PR DESCRIPTION
**Pull request by RH-sdavey**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/chronyc.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# chronyc
+
+> Query Chrony (NTP) daemon.
```
Comment (bl-ue) [inline pages/linux/chronyc.md:3]:
```suggestion
> Query the Chrony NTP daemon.
```

File: pages/linux/chronyc.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Use chronyc in interactive mode:
```
Comment (bl-ue) [inline pages/linux/chronyc.md:5]:
```suggestion
- Start chronyc in interactive mode:
```

File: pages/linux/chronyc.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`chronyc`
+
+- Display tracking stats for chrony daemon:
```
Comment (bl-ue) [inline pages/linux/chronyc.md:9]:
```suggestion
- Display tracking stats for the Chrony daemon:
```

File: pages/linux/chronyc.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`chronyc tracking`
+
+- Print time sources chrony is currently using as a time source:
```
Comment (bl-ue) [inline pages/linux/chronyc.md:13]:
```suggestion
- Print the time sources that Chrony is currently using as a time source:
```

File: pages/linux/ntpq.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# ntpq
+
+> Query Network Time Protocol (NTP) daemon.
```
Comment (bl-ue) [inline pages/linux/ntpq.md:3]:
```suggestion
> Query the Network Time Protocol (NTP) daemon.
```

File: pages/linux/ntpq.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- Use ntpq in interactive mode:
```
Comment (bl-ue) [inline pages/linux/ntpq.md:5]:
```suggestion
- Start ntpq in interactive mode:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Thank you for the new pages @RH-sdavey!  Will you please add links to them? https://manned.org/chronyc and https://manned.org/ntpq will work well if you can't find anything more canonical.

File: pages/linux/chronyc.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# chronyc
+
+> Query Chrony (NTP) daemon.
+
```
Comment (marchersimon) [inline pages/linux/chronyc.md:4]:
```suggestion
More information: <https://chrony.tuxfamily.org/doc/4.0/chronyc.html>.

```

File: pages/linux/ntpq.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# ntpq
+
+> Query Network Time Protocol (NTP) daemon.
+
```
Comment (marchersimon) [inline pages/linux/ntpq.md:4]:
```suggestion
More information: <https://www.eecis.udel.edu/~mills/ntp/html/ntpq.html>.

```

File: pages/linux/ntpq.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+- Use ntpq in interactive mode:
+
+`ntpq -i`
```
Comment (marchersimon) [inline pages/linux/ntpq.md:7]:
```suggestion
`ntpq --interactive`
```
For reference: the long options are neither listed in the official man page, nor by `ntpq --help`, but they are in <https://manned.org/ntpq> and they do work

File: pages/linux/ntpq.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+- Print list of ntp peers:
+
+`ntpq -p`
```
Comment (marchersimon) [inline pages/linux/ntpq.md:11]:
```suggestion
`ntpq --peers`
```

File: pages/linux/ntpq.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+- Print list of ntp peers without resolving hostnames from IP addresses:
+
+`ntpq -np`
```
Comment (marchersimon) [inline pages/linux/ntpq.md:15]:
```suggestion
`ntpq --numeric --peers`
```

File: pages/linux/ntpq.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+- Use ntpq in debugging mode:
+
+`ntpq -d`
```
Comment (marchersimon) [inline pages/linux/ntpq.md:19]:
```suggestion
`ntpq --debug-level`
```

File: pages/linux/ntpq.md [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+
+- Print ntp system variables values:
+
+`ntpq -c rv`
```
Comment (marchersimon) [inline pages/linux/ntpq.md:23]:
```suggestion
`ntpq --command={{rv}}`
```

**Review by marchersimon — CHANGES_REQUESTED** [review]

**Comment by bl-ue** [comment]
Whoa, that's a lot of errors.  Let me minimize those...

**Review by bl-ue — APPROVED** [review]

File: pages/linux/ntpq.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
+# ntpq
+
+> Query the Network Time Protocol (NTP) daemon.
+> More information: <https://www.eecis.udel.edu/~mills/ntp/html/ntpq.html>.
+
+- Start ntpq in interactive mode:
+
+`ntpq --interactive`
+
+- Print list of ntp peers:
```
Comment (marchersimon) [inline pages/linux/ntpq.md:10]:
```suggestion
- Print a list of ntp peers:
```
@bl-ue should we use ``` `ntp` ``` or `NTP` here?

File: pages/linux/ntpq.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+
+`ntpq --peers`
+
+- Print list of ntp peers without resolving hostnames from IP addresses:
```
Comment (marchersimon) [inline pages/linux/ntpq.md:14]:
```suggestion
- Print a list of ntp peers without resolving hostnames from IP addresses:
```

File: pages/linux/ntpq.md (hunk 14 above)
Comment (bl-ue) [reply pages/linux/ntpq.md:10]: Probably the latter, `NTP`.

File: pages/linux/chronyc.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+- Display stats for sources currently used by chrony daemon as a time source:
+
+`chronyc sourcestats`
+
+- Step the system clock immediately, bypassing any slewing:
+
+`chronyc makestep`
+
+- Display verbose information about each ntp source:
```
Comment (marchersimon) [inline pages/linux/chronyc.md:26]:
```suggestion
- Display verbose information about each NTP source:
```

File: pages/linux/ntpq.md [hunk 17]
```diff
@@ -0,0 +1,24 @@
+# ntpq
+
+> Query the Network Time Protocol (NTP) daemon.
+> More information: <https://www.eecis.udel.edu/~mills/ntp/html/ntpq.html>.
+
+- Start ntpq in interactive mode:
```
Comment (marchersimon) [inline pages/linux/ntpq.md:6]:
```suggestion
- Start `ntpq` in interactive mode:
```

File: pages/linux/ntpq.md [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+
+- Print list of ntp peers:
+
+`ntpq --peers`
+
+- Print list of ntp peers without resolving hostnames from IP addresses:
+
+`ntpq --numeric --peers`
+
+- Use ntpq in debugging mode:
```
Comment (marchersimon) [inline pages/linux/ntpq.md:18]:
```suggestion
- Use `ntpq` in debugging mode:
```

File: pages/linux/ntpq.md [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+
+`ntpq --debug-level`
+
+- Print ntp system variables values:
```
Comment (marchersimon) [inline pages/linux/ntpq.md:22]:
```suggestion
- Print NTP system variables values:
```

**Review by marchersimon — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**